src/osgPlugins/ive/ReadWrite.h

```diff
@@ -179,5 +192,12 @@ namespace ive {
 #define IVETEXT3D                       0x10000002
 #define IVEFADETEXT                     0x10000003
 
+class ReadWrite{
```

Comment:
We probably don't need this class added since we didn't modify it in the 3.4 makOsg.

Reply:
Removed the extraneous class.

Reply:
Update: this class is still in use:

src\osgplugins\ive\ExtendedMaterial.h(8): error C2504: 'ReadWrite': base class undefined
src\osgplugins\ive\AnimationMatrixTransform.h(12): error C2504: 'ReadWrite': base class undefined
src\osgplugins\ive\Animation.h(13): error C2504: 'ReadWrite': base class undefined